common/scripts/create-component.sh

```diff
@@ -114,6 +118,15 @@ for devenv in dev test ; do
         "--param=ENV=${devenv}"
         )
 
+    if [ ! -z "${QSBASE}" ]; then
```

Comment:
Wondering if it should fail or at least print a warning? I guess this is kind of a required param now?

Reply:
When not provided the quickstarter falls back to the default defined in the yml template. 

I would add a warning unless you feel it should be required. I am 50/50 on this. Let me know if you have a preference.

Reply:
I pushed a comment so this is now a required parameter. One point less variability reduces complexity. Thanks for the feedback
But internal quickstarters which use the scripts will now need to adjust, which should be fine as this is all new.